rust/processor/src/main.rs

```diff
@@ -10,7 +10,7 @@ const RUNTIME_WORKER_MULTIPLIER: usize = 2;
 
 fn main() -> Result<()> {
     let num_cpus = num_cpus::get();
-    let worker_threads = (num_cpus * RUNTIME_WORKER_MULTIPLIER).max(16);
+    let worker_threads = num_cpus * RUNTIME_WORKER_MULTIPLIER;
```

Comment:
not related to this PR, but why do we need a multiplier here?

Reply:
hyperthreading? IDK aptos-core had this I think

Reply:
`num_cpus::get()` actually considers hyperthreading.